app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,19 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self,
+                 nickname: str,
+                 musical_instrument: str,
+                 bow_level: int) -> None:
+
+        super().__init__(nickname=nickname,
+                         musical_instrument=musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return f"Elf ranger {self.nickname}. " \
```

Comment:
use () instead of \ here and all of the code

Reply:
Sorry, missed that.